augur/server.py

```diff
@@ -83,72 +73,32 @@ def addTimeseries(app, function, endpoint):
     app.route('/{}/<owner>/<repo>/timeseries/{}/relative_to/<ownerRelativeTo>/<repoRelativeTo>'.format(AUGUR_API_VERSION, endpoint))(flaskify(augur.util.makeRelative(function)))
 
 
+# Create Flask application
 app = Flask(__name__)
-CORS(app)# Try to open the config file and parse it
-parser = configparser.RawConfigParser()
```

Comment:
@howderek Are there changes we made in the past few days not reflected in @kmn5409 's commits ... this doesn't look right ... to delete this config checking stuff.